utilities/rpp-unittests/HIP_NEW/BatchPD_hip_pkd3.cpp

```diff
@@ -753,7 +753,8 @@ int main(int argc, char **argv)
     rppHandle_t handle;
     hipStream_t stream;
     hipStreamCreate(&stream);
-    rppCreateWithStreamAndBatchSize(&handle, stream, noOfImages);
+    RppBackend backend = RppBackend::RPP_HIP_BACKEND;
+    rppCreate(&handle, noOfImages, stream, backend);
```

Comment:
please change the parameter in original order and keep bachend as default parameter (setting it to HIP backend)

Reply:
resolved in discussion. new parameter order to be followed in MIVisionX